src/main/java/TaskList.java

```diff
@@ -0,0 +1,115 @@
+import java.util.Iterator;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * TaskList manages the list of tasks and implements the iterable interface
+ */
+public class TaskList implements Iterable<Task> {
+
+
+    public ArrayList<Task> listOfItems;
+
+
+    public ArrayList<Task> ListOfKeyWordItems = new ArrayList<>();
+
+    public String line = "____________________________________________________________";
```

Comment:
Maybe the line should belong to UI?

Reply:
Noted, it was left there from previous builds